src/main/java/com/compilercharisma/chameleonbusinessstudio/dto/validators/AppointmentValidator.java

```diff
@@ -1,5 +1,6 @@
-package com.compilercharisma.chameleonbusinessstudio.entity.appointment;
+package com.compilercharisma.chameleonbusinessstudio.dto.validators;
```

Comment:
I think Validators should have their own separate package, not inside the `/dto` package

Reply:
Things will get messy if we go with a flat structure, but you're the one with a job using Spring Boot, so I'll go with your judgement.